couchbase-operator/templates/sync-gateway-deployment.yaml

```diff
@@ -62,7 +62,7 @@ spec:
 {{ toYaml .Values.syncGateway.resources | indent 10 }}
       {{- if .Values.syncGateway.monitoring.prometheus.enabled }}
       - name: exporter
-        image: {{ default "couchbasesamples/sync-gateway-prometheus-exporter:latest" .Values.syncGateway.monitoring.prometheus.image }}
+        image: "{{ .Values.syncGateway.monitoring.prometheus.image.repository }}:{{ .Values.syncGateway.monitoring.prometheus.image.tag }}"
```

Comment:
Instead of double quote, can pipe you pipe this through the "| quote" template function.  I *think* what you have now may result in literal string.

Reply:
This has now been fixed